src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java

```diff
@@ -157,7 +156,7 @@ private static StringBuilder buildMessage(String address, String username,
                                 + "</s:Envelope>", schemaLocation, soapAction,
                                 guid, address,
                                 securityHeaderBuilder.toString(),
-                                rstTrustNamespace, DEFAULT_APPLIES_TO, keyType,
+                                rstTrustNamespace, cloudAudienceUrn, keyType,
```

Comment:
what if the cloudAudienceUrn is null? we should use default value in case of null.